src/link/core.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import type { ComponentProps, ComponentType } from 'react'
-import { Platform } from 'react-native'
+import { Platform, Linking } from 'react-native'
```

Comment:
Last thing: could we tree-shake `openUrl` in a separate file such that it only imports `Linking` on native?